store/localcachelayer/reaction_layer_test.go

```diff
@@ -28,10 +28,10 @@ func TestReactionStoreCache(t *testing.T) {
 		require.NoError(t, err)
 
 		reaction, err := cachedStore.Reaction().GetForPost("123", true)
-		require.Nil(t, err)
+		require.NoError(t, err)
 		assert.Equal(t, reaction, []*model.Reaction{&fakeReaction})
 		mockStore.Reaction().(*mocks.ReactionStore).AssertNumberOfCalls(t, "GetForPost", 1)
-		require.Nil(t, err)
+		require.NoError(t, err)
```

Comment:
Not sure if this line is actually necessary.

Reply:
Removed because you catch it, but the purpose of this PR is to migrate from the `require.Nil` to `require.NoError` for the `error` types, and keep the `Nil` for the `*model.AppError` types.